src/components/ChartTitle.jsx

```diff
@@ -13,44 +13,54 @@ const ChartTitle = ({
     isPriceUp,
     currentSymbol,
     Menu,
-    CategoricalDisplay,
+    MarketSelector,
```

Comment:
this is to remove eslint's shadow warnings. IMHO it also expresses intent more clearly